src/js/letters/containers/AddressSection.jsx

```diff
@@ -295,9 +301,16 @@ export class AddressSection extends React.Component {
           <div className="letters-address street">{streetAddress}</div>
           <div className="letters-address city-state">{cityStatePostal}</div>
           <div className="letters-address country">{country}</div>
+          <button className="address-help-btn" onClick={this.openAddressHelp}>What is This?</button>
```

Comment:
Should it be a lowercase `t` in `this`?

Reply:
Gah, good catch. The issue description does use a lowercase 't'.